src/Evolution/Initialization/InitialData.hpp

```diff
@@ -9,7 +9,7 @@
 #include "Utilities/Requires.hpp"
 #include "Utilities/TMPL.hpp"
 
-namespace evolution {
+namespace evolution ::initial_data {
```

Comment:
I believe @nilsdeppe meant to make this `evolution::Initialization`. There are a few places in the code that use this function, that you probably have to change to use the new namespace. (squash directly)

Reply:
yes, that's what I meant. Also, please clang-format (no space before `::`)

Reply:
Ooops. yep, that's right. My bad :joy: